index.js

```diff
@@ -257,10 +260,12 @@ function get20s(/*Your Code Here*/){
  
  For example, if removeArtist is invoked with the artists array and the number 0, it will remove Amedeo Modigliani from our dataset and return the number 19. */
 
-function removeArtist(/*Your Code Here*/){
-   /*Your Code Here*/
+function removeArtist(array, number){
+  artists.splice(0,number);
+  return(artists.length);
 }
-   
+console.log(removeArtist(artists, 1));
```

Comment:
What happens if you run the function like this:   removeArtist(artists, 0);   Does it remove Amedeo?